.env.example

```diff
@@ -1,3 +1,17 @@
 ROOT_LOGGING_LEVEL=debug
 APP_LOGGING_LEVEL=debug
 WEB_LOGGING_LEVEL=debug
+
+MONGO_HOST="pagopa-ecommerce-user-stats-service-mongo"
+MONGO_PORT=27017
+MONGO_USERNAME=admin
+MONGO_PASSWORD=password
+MONGO_SSL_ENABLED=false
+MONGO_MIN_POOL_SIZE=0
+MONGO_MAX_POOL_SIZE=20
+MONGO_MAX_IDLE_TIMEOUT_MS=60000
+MONGO_CONNECTION_TIMEOUT_MS=1000
+MONGO_SOCKET_TIMEOUT_MS=10000
+MONGO_SERVER_SELECTION_TIMEOUT_MS=2000
+MONGO_WAITING_QUEUE_MS=2000
+MONGO_HEARTBEAT_FREQUENCY_MS=5000
```

Comment:
add those variables to README and helm values too (for helm values you can use the same values as done [here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/d50c375fdfe411bfc6988f4dea95f4811e02b353/helm/values-dev.yaml#L101-L112) :)

Reply:
done here 1ca48baac7ea9bfc539cd80c60a6371868f6d721